docs/modules/schema/api-reference/table.md

```diff
@@ -1 +1,30 @@
 # Table
+
+loaders.gl defines a number of table types.
+
+- `ObjectRowTable`
+- `ArrayRowTable`
+- `GeoJSONTable`
+- `ColumnarTable`
+- `ArrowTable`
```

Comment:
question: how do we use the "binary geometry format" in these tables? should the loaders.gl return the binary geometries based on request?

Reply:
At the moment, the binary geometry format is not part of the table category, but a separate format supported by loaders.gl. 

We might consider changing that but would require additional utilities etc.

Regardless, they should all have a `shape` field that let's you tell which format you are dealing with.